api/metrics-api-jaxrs-1.1/pom.xml

```diff
@@ -124,6 +124,16 @@
   <build>
     <finalName>hawkular-metrics-api-rest-1.1</finalName>
     <plugins>
+      <plugin>
```

Comment:
The goal is to be able to do `mvn jboss-as:run` from this project and have the war running inside the container.  I will add the properties to the pom.